athena-federation-sdk-tools/src/main/java/com/amazonaws/athena/connector/validation/FederationService.java

```diff
@@ -1,30 +0,0 @@
-/*-
```

Comment:
Why is this file removed?

Reply:
Interface definitions have been removed for lambda v2, so FederationServiceProvider.java updated and interfaces removed.